challenges/10exercises.js

```diff
@@ -1,26 +1,109 @@
 // Define a function max() that takes two numbers as arguments and returns the largest of them.
 // Use the if-then-else construct available in Javascript.
 
+function max( value1, value2 ) {
+  if ( value1 > value2 ) {
+    return value1
+  }
+  else {
+    return value2
+  }
+}
```

Comment:
What if the numbers are equal to each other? I would just add a line for that possibility.